dataprofiler/profilers/profiler_options.py

```diff
@@ -599,8 +599,55 @@ def _validate_helper(self, variable_path='TextOptions'):
             errors.append("{}.vocab must be a BooleanOption."
                           .format(variable_path))
         errors += self.vocab._validate_helper(variable_path + '.vocab')
+
+        if self.properties["num_zeros"].is_enabled:
+            errors.append("{}.num_zeros should always be disabled,"
+                          " num_zeros.is_enabled = False"
+                          .format(variable_path))
+
+        if self.properties["num_negatives"].is_enabled:
+            errors.append("{}.num_negatives should always be disabled,"
+                          " num_negatives.is_enabled = False"
+                          .format(variable_path))
         return errors
 
+    @property
+    def is_numeric_stats_enabled(self):
+        """
+        Returns the state of numeric stats being enabled / disabled. If any
+        numeric stats property is enabled it will return True, otherwise it
+        will return False.
+
+        :return: true if any numeric stats property is enabled, otherwise false
+        :rtype bool:
+        """
+        if self.min.is_enabled or self.max.is_enabled or self.sum.is_enabled \
+                or self.variance.is_enabled or self.skewness.is_enabled \
+                or self.kurtosis.is_enabled or self.bias_correction.is_enabled \
+                or self.histogram_and_quantiles.is_enabled \
+                or self.num_zeros.is_enabled or self.num_negatives.is_enabled:
```

Comment:
If these 2 are always disabled should they even be present here? Also why can't we just use the one in `NumericOptions` here? I understand the different setter since you want to set differently in this case, but this property seems identical to the `NumericOptions` one.